cmd/clusterctl/pkg/internal/util/yaml.go

```diff
@@ -52,11 +54,11 @@ func JoinYaml(yamls ...[]byte) []byte {
 // ToUnstructured takes a YAML and converts it to a list of Unstructured objects
 func ToUnstructured(rawyaml []byte) ([]unstructured.Unstructured, error) {
 	var ret []unstructured.Unstructured //nolint
-	for _, data := range bytes.Split(rawyaml, yamlSeparator) {
+	for _, data := range yamlSeparatorRegEx.Split(string(rawyaml), -1) {
```

Comment:
Do we actually need to `Split` here? We can make use of the https://godoc.org/k8s.io/apimachinery/pkg/util/yaml#YAMLDecoder to automatically split the incoming objects.

Reply:
done!